linkerd/app/outbound/src/target.rs

```diff
@@ -13,6 +13,7 @@ pub struct EndpointFromMetadata;
 #[derive(Copy, Clone, Debug, Eq, PartialEq, Hash)]
 pub struct Accept<P> {
     pub orig_dst: SocketAddr,
+    pub local_addr: SocketAddr,
```

Comment:
Can this be removed now?

Reply:
whoops, missed that one. should be good now!